engine/apps/user_management/sync.py

```diff
@@ -71,27 +70,44 @@ def _sync_instance_info(organization):
         organization.gcom_token_org_last_time_synced = timezone.now()
 
 
-def sync_users_and_teams(client, api_users, organization):
+def sync_users_and_teams(client: GrafanaAPIClient, organization):
+    sync_users(client, organization)
+    sync_teams(client, organization)
+    sync_team_members(client, organization)
+
+    organization.last_time_synced = timezone.now()
```

Comment:
nit; would suggest adding this on the line under the call to `sync_users_and_teams` rather than within this method